content/starter-kits/ethathon.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "ETHathon Full-Stack DApp Boilerplate"
+description: "Highly-integrated, fast, and type-safe monorepo with NextJS, Tailwind, Hardhat, wagmi.sh, and Rainbowkit."
+authors: ["@dennis_zoma"]
+tags: ["Smart Contracts","Dapp","Monorepo"]
```

Comment:
Can you please remove `monorepo` from the tags?

Other than that, looks good to me! Cool project

Reply:
thanks! just removed it :)